pythonforandroid/recipes/libxslt/__init__.py

```diff
@@ -24,7 +24,8 @@ def build_arch(self, arch):
             env["CC"] += " -I%s" % self.get_build_dir(arch.arch)
             libxml = (
                 dirname(dirname(self.get_build_container_dir(arch.arch)))
-                + "/libxml2/%s/libxml2" % arch.arch
+                + "/libxml2/%s/libxml2" % basename(
+                    self.get_build_container_dir(arch.arch))
```

Comment:
I think lines 25 to 28 can just be `libxml = self.get_build_dir(arch.arch)`. Am I missing something here, since the rest of the recipe uses `get_build_dir` normally I don't see why this part was originally written differently?

Reply:
Oh, I see. In that case you can do `Recipe.get_recipe('libxml2', self.ctx).get_build_dir(arch.arch)`. This has the advantage of being fully future proof.

Reply:
I added in your suggested change! <3